appium/webdriver/extensions/flutter_integration/scroll_directions.py

```diff
@@ -0,0 +1,8 @@
+from enum import Enum
+
+class ScrollDirection(Enum):
+    UP = 'up'
+    DOWN = 'down'
+    
+    def as_string(self):
```

Comment:
this method is not really needed. You may use `.value`

Reply:
Thanks! fixed

Reply:
was the change committed?

Reply:
Yes .. I forgot to push the changes